dynamicRooms/README.md

```diff
@@ -0,0 +1,21 @@
+# RSCBot: combineRooms
+
+The `dynamicRooms` cog is primarily responsible for enabling automatic voice channel management based server usage. This involves dynamically creating and deleting voice channels under a set category.
+
+## Installation
+
+The `dynamicRooms` cog is independant from all other cogs. No prerequisites are required.
+
+```
+<p>cog install RSCBot combineRooms
+<p>load combineRooms
+```
+
+## Usage
+
+- `<p>addDynamicCategory`
+  Sets all voice channels contained by a category to become dynamic voice channels.
```

Comment:
I think you should explain what it means to be a dynamic voice channel here and in the help section of the command.

Reply:
will do.

Reply:
done.